src/Forms/Main.cs

```diff
@@ -22350,16 +22350,23 @@ private void ToolStripMenuItemGoogleMicrosoftTranslateSelLineClick(object sender
                 var p = _subtitle.GetParagraphOrDefault(firstSelectedIndex);
                 if (p != null)
                 {
-                    string defaultFromLanguage = LanguageAutoDetect.AutoDetectGoogleLanguage(_subtitle);
+                    string defaultFromLanguage = string.Empty;

```

Comment:
don't run the auto-detect language twice - if we are giving more priority to alternate-subtitle